.github/workflows/add-pr-label.yaml

```diff
@@ -0,0 +1,17 @@
+name: PR Auto Labeling
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
we should change this temporarily to confirm that it works, then put it back to only trigger on opened pull request events (rather than any pr event, i.e. `pull_request: {}`)

Reply:
Good call. Looks like it worked :)